src/library_sdl.js

```diff
@@ -15,12 +15,21 @@
 //                   or otherwise).
 
 var LibrarySDL = {
+  $SDL_unicode__docs: '/** @suppress{missingProperties} */',
+  $SDL_unicode: function() { return SDL.unicode},
+
+  $SDL_ttfContext__docs: '/** @suppress{missingProperties} */',
+  $SDL_ttfContext: function() { return SDL.ttfContext},
+
+  $SDL_audio__docs: '/** @suppress{missingProperties} */',
+  $SDL_audio: function() { return SDL.audio},
```

Comment:
why not just put these 3 properties on the `SDL` object? the functions approach seems larger?

Reply:
That is because `SDL.audio__docs: '/** @suppress{missingProperties} */',` would not work. We can only annotate top level symbols.

Reply:
I see, thanks.